bots/tests-invoke

```diff
@@ -48,6 +48,7 @@ def main():
             help="Work offline, don''t fetch new data from origin for rebase")
     parser.add_argument('--publish', dest='publish', default=os.environ.get("TEST_PUBLISH", ""),
             action='store', help='Publish results centrally to a sink')
+    parser.add_argument('--updater', help='Hook that runs before the tests')
```

Comment:
Totally bikeshedding of course, but how about calling it `--pre-test`? We might get other hooks in the future, like for collecting additional state/logs after the test, and introduce a `--post-hook` then.